lerna.json

```diff
@@ -1,11 +1,5 @@
 {
     "version": "2.0.0-rc.17",
-    "packages": [
-        "packages/peregrine",
-        "packages/pwa-buildpack",
-        "packages/upward-js",
-        "packages/upward-spec",
-        "packages/venia-concept"
-    ],
-    "npmClient": "npm"
+    "npmClient": "yarn",
+    "useWorkspaces": true
```

Comment:
can we just remove this file now that we are going to use yarn workspaces?

Reply:
looks like lerna.json is obsolete now, same question as above, can we remove this file now? It looks like you removed lerna from everywhere so far but [this file](https://github.com/magento-research/pwa-studio/blob/release/2.0/scripts/watch-all.js#L180). We probably want to make that change as well.